compile.js

```diff
@@ -6,7 +6,7 @@ creates a highly compressed release build in bin of the contents of src
 
 packages used:
 
-npm i rimraf compress-images web-resource-inliner ncp gifsicle@5.3.0 concat ycssmin terser gzipper html-minifier-terser glob   
+npm i rimraf compress-images web-resource-inliner ncp gifsicle@5.3.0 concat ycssmin terser gzipper html-minifier-terser glob@8   
```

Comment:
Should probably remove this line (and maybe replace it with another line saying to use `npm install` to install the packages listed in `package.json`.